### PR TITLE
fix(work_order): Add item name in Work Order

### PR DESCRIPTION
**Pull request by Alchez**
**Refs:**

- https://corp.bloomstack.garden/desk#Form/Issue/ISS-2019-00009
- https://github.com/DigiThinkIT/delta9/pull/96

<hr>

**Patch:**

image

**Form View:**

image

### CODE REVIEW

File: erpnext/manufacturing/doctype/work_order/work_order.json [hunk 1]
```diff
@@ -1,1732 +1,1765 @@
 {
```
Comment (vjFaLk) [inline erpnext/manufacturing/doctype/work_order/work_order.json:1]: Pls fix

**Review by vjFaLk — CHANGES_REQUESTED** [review]

File: erpnext/manufacturing/doctype/work_order/work_order.json (hunk 1 above)
Comment (Alchez) [reply erpnext/manufacturing/doctype/work_order/work_order.json:1]: The spaces are actually ERPNext convention (don't ask), so this is actually a fix. :man_facepalming:

**Comment by vjFaLk** [comment]
Fix conflict and make to upstream

**Comment by Alchez** [comment]
Made upstream at https://github.com/frappe/erpnext/pull/16608.

**Pull request merged**